api/agentpb/agent.go

```diff
@@ -47,6 +49,12 @@ type AgentConnectMetadata struct {
 	Version string
 }
 
+// AgentServerMetadata represents metadata sent by pmm-managed to pmm-agent.
+type AgentServerMetadata struct {
+	RunsOnNodeID      string
```

Comment:
Maybe `AgentRunsOnNodeID`? To make it clear that is Agent's Node, not PMM Server's Node.